src/component/JoinList.js

```diff
@@ -63,8 +61,8 @@ const JoinList = ({ firestore }) => {
           Join List
         </button>
       </form>
-
-      <p>Nevermind! <a href="/" className="newListLink">Start a New List</a></p>
+      <p> - OR - </p>
```

Comment:
Semantically speaking this is not a _real_ paragraph 😬, I think it would be good to wrap this in a `span`, or a `div` instead. What do you think about that?

Reply:
Yeah, I didn't think about that. Do you know of any other alternatives, because even `span` and `div` seem unhelpful. Is there an `aside` tag or anything like that? What about an `H3`?

Reply:
Good question!

I don't think that `aside` will do the trick because you generally use it in sidebars. [docs about aside](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside)

An H3 won't be correct either because this ` - OR - ` is not a title per se.

That said, I can't think about anything different than a `span` or a `div` :/

Reply:
If you need a wrapper for this bit of text that is purely for styling (i.e. no semantics involved) then a `<span>` is perfectly appropriate. What I might suggest, though, is wrapping the text before and after the "- OR -" in the same `<p></p>` tag and including your new `<span></span>` within it as well. In other words, treat the paragraph as one paragraph, which it is semantically

Reply:
Change made! 

![bitmoji](https://render.bitstrips.com/v2/cpanel/5b7bff30-d7cc-4141-9f91-d77fa135057f-141935a4-509e-461c-bba3-4228c119c4d3-v1.png?transparent=1&palette=1&width=100)